### DISCUSSION TITLE
Register colima as MacOS Service

### DISCUSSION THREAD

**Discussion by earlbw** [abiosoft/colima#238] [Q&A] [answered]
What is the easiest/best way to register colima as service to auto start/stop VMs?  I've shutdown my Mac without stopping colima VM(s) from the command line and would like a way to automatically shut them down when I forget to do that.

**Comment by rfay** [comment, marked as answer]
I think the answer will be in 
* https://github.com/abiosoft/colima/issues/96
when it comes.
